couch2pg/src/importer.js

```diff
@@ -131,8 +131,9 @@ const storeDocs = async (allDocsResult) => {
 };
 
 const importChangesBatch = async (couchDb, source) => {
+  const dbName = couchDb.name.split('/').pop();
   const seq = await getSeq(source);
-  console.info('Downloading CouchDB changes feed from ' + seq);
+  console.info(`Downloading CouchDB changes feed from ${seq} in ${dbName}`);
```

Comment:
Can you please format the message like `Downloading ${dbName} changes feed from ${seq}` instead? 
Obviously we're syncing from Couch (omg) and the seq is super long and it will be really hard to eye-scan for the end of the interminable string to spot the dbname.